
---
int32 dirAntMaxDir
int32 dirAntMaxTilt
int32 dirAntLevel

